src/flb_input_chunk.c

```diff
@@ -1231,8 +1240,11 @@ int flb_input_chunk_append_raw(struct flb_input_instance *in,
     int new_chunk = FLB_FALSE;
     uint64_t ts;
     size_t diff;
```

Comment:
It seems like your version of the accounting code renders this variable and the code blocks used to calculate its contents obsolete so let's remove all that to make the logic easier to understand

Reply:
I redid the accounting code changing the name of size to content_size for the few checks we still need it for. Please review it.